osc-server/src/test/java/org/osc/core/broker/util/network/NetworkSettingsApiTest.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.util.network;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Matchers;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.osc.core.broker.util.ServerUtil;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+//TODO balmukund: The class under test now powermock is being used to mock static dependencies. 
+//This will be removed when refactoring happens along with other classes.
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ ServerUtil.class, Runtime.class })
+public class NetworkSettingsApiTest {
+
+    private NetworkSettingsApi networkSettingsApi;
+    private Runtime runtimeMock;
+    private Process processMock;
+
+
+    @Before
+    public void init() throws Exception {
+        MockitoAnnotations.initMocks(this);
+        networkSettingsApi = new NetworkSettingsApi();
+        PowerMockito.mockStatic(Runtime.class);
+        runtimeMock = PowerMockito.mock(Runtime.class);
```

Comment:
you should not need all these mocks. Just mocking the PowerMockito.when(ServerUtil.execWithLines(Matchers.anyString(), Matchers.anyList())) to return specific values should be sufficient.